ts/screens/wallet/TextVerificationScreen.tsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * This screen allows the user to input the code
+ * received via text from PagoPA
+ */
+import {
+  Body,
+  Button,
+  Container,
+  Content,
+  H1,
+  Input,
+  Left,
+  Text,
+  View
+} from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import { Col, Grid } from "react-native-easy-grid";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import { WalletAPI } from "../../api/wallet/wallet-api";
+import AppHeader from "../../components/ui/AppHeader";
+import IconFont from "../../components/ui/IconFont";
+import PaymentBannerComponent from "../../components/wallet/PaymentBannerComponent";
+import I18n from "../../i18n";
+import ROUTES from "../../navigation/routes";
+import variables from "../../theme/variables";
+import { TransactionSummary } from "../../types/wallet";
+
+type Props = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+
+const styles = StyleSheet.create({
+  contentPadding: {
+    paddingRight: variables.contentPadding,
+    paddingLeft: variables.contentPadding
+  },
+  textRight: {
+    textAlign: "right"
+  },
+  bottomBordered: {
+    borderBottomWidth: 1
+  }
+});
+
+export class TextVerificationScreen extends React.Component<Props> {
+  constructor(props: Props) {
+    super(props);
+  }
+
+  public render(): React.ReactNode {
+    const transaction: Readonly<
+      TransactionSummary
+    > = WalletAPI.getTransactionSummary();
```

Comment:
I guess this will be fetched from redux?

Reply:
Absolutely yes -- all those `WalletAPI`-based calls will be redux-based